lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
+
+        parser.on('--force-color', '--force-colour', 'Force the output to be in color.') do |_o|
+          options[:force_color] = true
+          options.delete(:no_color)
+        end
+
+        parser.on('--no-color', '--no-colour', 'Force the output to not be in color.') do |_o|
+          options[:no_color] = true
+          options.delete(:force_color)
         end
```

Comment:
It'd be nice to find a way to list these two options as one (particularly if we are dropping `--color` from `--help`) similar to the `--[no-]color` thing we have now -- although I'm not sure what the standard is for CLI tools when you are trying to show `--force-color` or `--no-color`.  If there's no good way to combine them in the output we can keep them separate.

Regardless, it'd be nice if the description mentioned something like:

* `Force color on or off, ignoring TTY status of output` (e.g. if we combine them)
* `Force the output to be in color, even if the output is not a TTY` (for `--force-color`)
* `Force the output to not be in color, even if the output is a TTY` (for `--no-color`)